CHANGELOG

```diff
@@ -70,6 +70,14 @@ v3.10.6 (XXXX-XX-XX)
 v3.10.5 (2023-03-16)
 --------------------
 
+* BTS-1148: Fix a race when aborting/finishing a currently active query
```

Comment:
This should now go into the 3.10.6 section of the CHANGELOG, as 3.10.5 has already been released by now.